homeassistant/components/zha/light.py

```diff
@@ -1023,6 +1051,22 @@ def __init__(
         """Initialize a light group."""
         super().__init__(entity_ids, unique_id, group_id, zha_device, **kwargs)
         group = self.zha_device.gateway.get_group(self._group_id)
+
+        # ugly experimental code to support "execute if off" for groups
+        # Note: Can this be done in a better way?
+        # Note: Also, currently, this is only set once when the group is initialized,
+        #       so if the "options" attribute is changed, ZHA needs to be reloaded.
+        self._GROUP_SUPPORTS_EXECUTE_IF_OFF = True  # pylint: disable=invalid-name
+        for member in group.members:
+            for pool in member.device.channels.pools:
+                for channel in pool.all_channels.values():
+                    if (
+                        channel.name == CHANNEL_COLOR
+                        and not channel.execute_if_off_supported
+                    ):
+                        self._GROUP_SUPPORTS_EXECUTE_IF_OFF = False
+                        break
```

Comment:
Is there a *simple way* for a ``LightGroup`` to check if all members have a certain attribute/property set?

The above code "works", but it's not really clean of course.